2023-10-02 19:49:21 - INFO - [base_logger.py : create_logger_for_service - 30] :: Initialized logger for the service [usermanagement] :: [SUCCESS]
2023-10-02 19:49:21 - INFO - [create_app.py : create_app_instance - 62] :: Received Service [usermanagement]
2023-10-02 19:49:21 - INFO - [create_app.py : create_app_instance - 64] :: Flask of Application Instance created for service [usermanagement] ,  App Name :: [<Flask 'usermanagement'>] ==> SUCCESS
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 339] :: Received Schema File Path : /Users/kamalsaidevarapalli/Desktop/Workshop/PyPortalAdminstration/src/usermanagement_service/schemas/headers/reg_headers_schema.json
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 352] :: Schema Validation True
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 339] :: Received Schema File Path : /Users/kamalsaidevarapalli/Desktop/Workshop/PyPortalAdminstration/src/usermanagement_service/schemas/headers/getuser_headers_schema.json
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 352] :: Schema Validation True
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 339] :: Received Schema File Path : /Users/kamalsaidevarapalli/Desktop/Workshop/PyPortalAdminstration/src/usermanagement_service/schemas/headers/del_user_headers_schema.json
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 352] :: Schema Validation True
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 339] :: Received Schema File Path : /Users/kamalsaidevarapalli/Desktop/Workshop/PyPortalAdminstration/src/usermanagement_service/schemas/requests/register_user/req_schema.json
2023-10-02 19:49:21 - INFO - [create_app.py : read_json_schema - 352] :: Schema Validation True
2023-10-02 19:49:21 - INFO - [create_app.py : create_blueprint - 83] :: Created Blueprint :: <Blueprint 'usermanagement_bp'> for the Application service ::usermanagement
2023-10-02 19:49:21 - INFO - [create_app.py : display_registered_blueprints_for_service - 122] :: Added Blueprints :: /static/<path:filename>
2023-10-02 19:49:21 - INFO - [create_app.py : create_db_engine - 145] :: Creating DB-Engine using DatabaseURI for Service :: [usermanagement]
2023-10-02 19:49:21 - INFO - [create_app.py : get_database_uri - 131] :: Preparing DatabaseURI for Service :: [usermanagement]
2023-10-02 19:49:21 - INFO - [create_app.py : get_database_uri - 134] :: Prepared DatabaseURI for Service :: [usermanagement] - postgresql://usermanageradmin:Ishi%2306041995@127.0.0.1:3304/USERS
2023-10-02 19:49:21 - INFO - [create_app.py : create_db_engine - 150] :: Created DB-Engine using DatabaseURI for Service :: [Engine(postgresql://usermanageradmin:***@127.0.0.1:3304/USERS)]
2023-10-02 19:49:21 - INFO - [create_app.py : check_database_connectivity - 285] :: Current Connection status set to :: False for the service :: usermanagement
2023-10-02 19:49:21 - INFO - [create_app.py : check_database_connectivity - 286] :: Trying to establish the connection to the database :: [IN-PROGRESS]
2023-10-02 19:49:21 - INFO - [create_app.py : check_database_connectivity - 290] :: Established the connection to the database :: [SUCCESS]
2023-10-02 19:49:21 - INFO - [create_app.py : check_database_connectivity - 292] :: Current Connection status set to :: True for the service :: usermanagement
2023-10-02 19:49:21 - INFO - [create_app.py : check_database_connectivity - 296] :: Closing the Current Connection as the connection was established for the service :: usermanagement
2023-10-02 19:49:21 - INFO - [create_app.py : init_databases_for_service - 331] :: Database already exists :: USERS
2023-10-02 19:49:21 - INFO - [create_app.py : create_tables_associated_to_db_model - 307] :: Trying to create the tables if not present in the database...
2023-10-02 19:49:21 - INFO - [create_app.py : create_tables_associated_to_db_model - 309] :: Going to create the tables ...
2023-10-02 19:49:21 - INFO - [create_app.py : create_tables_associated_to_db_model - 312] :: Created the tables ...
2023-10-02 19:49:21 - INFO - [create_app.py : bind_db_app - 171] :: Binding SQLALCHEMY using DatabaseURI:: [postgresql://usermanageradmin:Ishi%2306041995@127.0.0.1:3304/USERS] to Application Instance for Service :: [usermanagement]
2023-10-02 19:49:21 - INFO - [create_app.py : bind_db_app - 175] :: Bound SQLALCHEMY :: [<SQLAlchemy>] to Application Instance for Service :: [usermanagement]
2023-10-02 19:49:21 - INFO - [create_app.py : migrate_db_bind_app - 185] :: Binding the db :: [<SQLAlchemy>] to app instance :: [<Flask 'usermanagement'>] for migrations
2023-10-02 19:49:21 - INFO - [create_app.py : migrate_db_bind_app - 188] :: Bound the db :: [<SQLAlchemy>] to app instance :: [<Flask 'usermanagement'>] for migrations
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 197] :: Creating Pool of connections for Service ==>[usermanagement] :: [STARTED]
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 199] :: Created Pool of connections for Service ==>[usermanagement] :: [SUCCESS]
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 200] :: Creating session using pool of connections  :: [STARTED]
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 201] :: Creating  a session maker for database interactions
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 203] :: Created session using Pool_info :: <sqlalchemy.pool.impl.QueuePool object at 0x104dd41c0>
2023-10-02 19:49:21 - INFO - [create_app.py : create_pool_of_connections - 204] :: Initialised a session maker for database interactions sessionmaker(class_='Session', bind=Engine(postgresql://usermanageradmin:***@127.0.0.1:3304/USERS), autoflush=True, expire_on_commit=True)
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 257] :: #---------------------------------[ POOL - INFO ]----------------------------------------#
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 258] :: Displaying Pool_Info for Service ==>[usermanagement] 
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 259] :: Current Pool Info :: <sqlalchemy.pool.impl.QueuePool object at 0x104dd41c0> - ID: 4376576448
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 260] :: Current Pool Size ::  30
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 261] :: Checked Out Connections from Pool  0
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 262] :: Checked in Connections available in Pool :: 0
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 263] :: Current Pool Overflow Info :: -30
2023-10-02 19:49:21 - INFO - [create_app.py : display_pool_info - 264] :: #---------------------------------[ POOL - INFO ]----------------------------------------#
2023-10-02 19:49:21 - INFO - [create_app.py : register_blueprint - 93] :: Registering blueprints to the service :: <Flask 'usermanagement'>
2023-10-02 19:49:21 - INFO - [create_app.py : display_registered_blueprints_for_service - 122] :: Added Blueprints :: /static/<path:filename>
2023-10-02 19:49:21 - INFO - [create_app.py : display_registered_blueprints_for_service - 122] :: Added Blueprints :: /api/v1/airliner/registerUser
2023-10-02 19:49:21 - INFO - [create_app.py : display_registered_blueprints_for_service - 122] :: Added Blueprints :: /api/v1/airliner/getUser/<int:userid>
2023-10-02 19:49:21 - INFO - [create_app.py : display_registered_blueprints_for_service - 122] :: Added Blueprints :: /api/v1/airliner/deleteUser/<int:userid>
2023-10-02 19:49:21 - INFO - [__init__.py : <module> - 133] :: Created GRPC server with the workers of max :: 10
2023-10-02 19:49:21 - INFO - [__init__.py : <module> - 135] :: Registered GRPC server to the server :: UserValidationForTokenGenerationService
2023-10-02 19:49:21 - INFO - [__init__.py : <module> - 137] :: Starting GRPC server for the Token-User service with the IP & PORT:: 0.0.0.0:9092
2023-10-02 19:49:21 - INFO - [usermanagement.py : <module> - 7] :: Started the grpc server ...
2023-10-02 19:49:21 - INFO - [usermanagement.py : <module> - 8] :: Started the USER-MANAGEMENT server ...
2023-10-02 19:49:21 - INFO - [usermanagement.py : <module> - 9] :: Binded USER-MANAGEMENT-SERVICE at IP-ADDRESS:PORT :: 0.0.0.0:9091
2023-10-02 19:49:21 - INFO - [usermanagement.py : <module> - 10] :: Application is ready to server traffic.
